server/odc-service/src/main/java/com/oceanbase/odc/service/structurecompare/comparedbobject/AbstractDBObjectStructureComparator.java

```diff
@@ -32,10 +32,15 @@ public abstract class AbstractDBObjectStructureComparator<T extends DBObject>
         implements DBObjectStructureComparator<T> {
     protected final String srcSchemaName;
     protected final String tgtSchemaName;
+    protected final String srcTableName;
+    protected final String tgtTableName;
```

Comment:
It is recommended to change these 2 fields to：
`projceted final List<String> srcObjectNames;`
`projceted final List<String> tgtObjectNames;`